### PR TITLE
MAP-645 Add ADR documents and rational

### PR DESCRIPTION
**Pull request by Mjwillis**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import uk.gov.justice.digital.hmpps.gradle.PortForwardRedisTask
 import uk.gov.justice.digital.hmpps.gradle.RevealSecretsTask
 
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "5.13.0"
```
Comment (Mjwillis) [inline build.gradle.kts:7]: Upgrades

File: docs/0004-location-db-design.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+| HOU_SANI_FIT | ACB        | Auditable Cell Bell                    | ACB  |
+| HOU_SANI_FIT | ABD        | Anti Barricade Door                    | ABD  |
+| HOU_SANI_FIT | FIB	       | Fixed Bed                              | FIB  |
+| HOU_SANI_FIT | MD	        | Metal Door                             | MD   |
+| HOU_SANI_FIT | MOB	       | Movable Bed                            | MOB  |
+| HOU_SANI_FIT | PC	        | Privacy Curtain                        | PC   |
+| HOU_SANI_FIT | PS	        | Privacy Screen                         | PS   |
+| HOU_SANI_FIT | SCB	       | Standard Cell Bell                     | SCB  |
+| HOU_SANI_FIT | SETO       | 	Separate Toilet                       | SETO |
+| HOU_SANI_FIT | WD	        | Wooden Door                            | XXX  |
```
Comment (Mjwillis) [inline docs/0004-location-db-design.md:189]: Will add other in next PR!

File: src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/Location.kt [hunk 3]
```diff
@@ -247,6 +247,7 @@ enum class LocationType(
   HOLDING_AREA("Holding Area"),
   BOOTH("Booth"),
   BOX("Box"),
+  ROOM("Room"),
```
Comment (Mjwillis) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/Location.kt:250]: This was missing and spotted when migrating real data.

**Review by jimbali — APPROVED** [review]
Nice docs!

**Pull request merged**